.github/workflows/deploy-to-oss.yaml

```diff
@@ -0,0 +1,44 @@
+name: Deploy Artifacts to OSS
+
+on:
+  push:
+    tags: [ "v*" ]
```

Comment:
on:
  push:
    # Sequence of patterns matched against refs/tags
    tags:        
      - "v*.*.*"

This is better, otherwise this workflow will be triggered unexpectedly when tag just starts with v.

Reply:
Done.